### ISSUE TITLE
Keypoint prior loss function

### ISSUE THREAD

**Issue by hanweikung** [zhengkw18/face-vid2vid#7]
Thank you for your work. May I ask why your keypoint prior loss function is slightly different from the one in [the original paper](https://nvlabs.github.io/face-vid2vid/main.pdf)?

In the paper (A.2), the keypoint prior loss function is:

![Screenshot from 2021-12-16 16-02-13](https://user-images.githubusercontent.com/643825/146332727-19086090-9715-4063-b3a5-1cfe1792b5eb.png)

However, yours in losses.py is:

```python
loss = (
    torch.max(0 * dist_mat, self.Dt - dist_mat).sum((1, 2)).mean()
    + torch.abs(kp_d[:, :, 2].mean(1) - self.zt).mean()
    - kp_d.shape[1] * self.Dt
)
```

I was wondering why you subtracted `kp_d.shape[1] * self.Dt` in the end.

**Comment by zhengkw18** [comment]
For the distance matrix `dist_mat`, its diagonal is zero. So `self.Dt - dist_mat` produced many `self.Dt` in its diagonal, which are summed into the loss and need to be thrown away.

**Comment by hanweikung** [comment]
I see now. Thank you for your prompt answer!
